SFI Wave 4 Remediation

v-thepet opened the pull request:
This PR is part of the effort to remove secure information from images. 4 images edited in AspNetDocs

For request details, see [360964](https://dev.azure.com/msft-skilling/Content/_workitems/edit/360964)

<!--
# Instructions

When creating a new PR, please reference the issue number if there is one:

Fixes #Issue_Number

The "Fixes #nnn" syntax in the PR description allows GitHub to automatically close the issue when this PR is merged.

NOTE: This is a comment; please type your descriptions above or below it.
-->

v-thepet commented:
Hi @Rick-Anderson - The Aquent Q&M team is assisting in remediation of flagged issues from the most recent SFI scan. Please review the image changes in this PR and approve and sign off, or request alternate changes that will meet current guidance. Thanks!

Rick-Anderson reviewed (approved):
@v-thepet  what was changed and what SFI violation? cc @tdykstra

v-thepet commented:
Hi @Rick-Anderson and @tdykstra - Here are the images, descriptions from my spreadsheet, and what I changed. The thumbprint was obvious, the others I guessed at what was possibly insecure or disallowed.

https://github.com/dotnet/AspNetDocs/blob/main/aspnet/web-api/overview/older-versions/build-restful-apis-with-aspnet-web-api/_static/image53.png. Description: The image shows a web application publish dialog with fields for Service URL, Site/application, User name, Password, and Destination URL. What I changed: Server name in the Service URL to something generic.
https://github.com/dotnet/AspNetDocs/blob/main/aspnet/web-forms/overview/getting-started/getting-started-with-aspnet-45-web-forms/checkout-and-payment-with-paypal/_static/image7.png. Description: The image contains a security warning about installing a certificate with a visible thumbprint. What I changed: Changed the thumbprint to an approved fictitious value per https://review.learn.microsoft.com/en-us/help/platform/reference-sensitive-identifier?branch=main#replacing-non-guid-identifiers-in-your-content.
https://github.com/dotnet/AspNetDocs/blob/main/aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/deploying-your-site-using-visual-studio-cs/_static/image13.png. Description: The image shows a file transfer interface with a connection string visible. What I changed: Changed the FTP address to something generic.
https://github.com/dotnet/AspNetDocs/blob/main/aspnet/web-forms/overview/older-versions-getting-started/deploying-web-site-projects/deploying-your-site-using-visual-studio-cs/_static/image15.png. Description: The image shows a file transfer interface with a connection string visible. What I changed: Changed the FTP address to something generic.

Rick-Anderson commented:
@v-thepet  thanks! Your changes are live now.
